commands/import.go

```diff
@@ -0,0 +1,251 @@
+}
+
+func importUploadCmdF(c client.Client, command *cobra.Command, args []string) error {
+	filepath := args[0]
+
+	file, err := os.Open(filepath)
+	if err != nil {
+		return fmt.Errorf("failed to open import file: %w", err)
+	}
+	defer file.Close()
+
+	info, err := file.Stat()
+	if err != nil {
+		return fmt.Errorf("failed to stat import file: %w", err)
+	}
+
+	shouldResume, _ := command.Flags().GetBool("resume")
+	var us *model.UploadSession
+	var resp *model.Response
+	if shouldResume {
+		uploadID, err := command.Flags().GetString("upload")
+		if err != nil || !model.IsValidId(uploadID) {
+			return errors.New("required upload ID is missing or invalid")
+		}
+
+		us, resp = c.GetUpload(uploadID)
+		if resp.Error != nil {
+			return fmt.Errorf("failed to get upload: %w", resp.Error)
+		}
+
+		if us.FileSize != info.Size() {
+			return fmt.Errorf("file sizes do not match")
+		}
+
+		if _, err := file.Seek(us.FileOffset, io.SeekStart); err != nil {
+			return fmt.Errorf("failed to get seek file: %w", err)
+		}
+	} else {
+		isLocal, _ := command.Flags().GetBool("local")
+		userID := "me"
+		if isLocal {
+			userID = fakeUserID
+		}
+
+		us, resp = c.CreateUpload(&model.UploadSession{
+			Filename: info.Name(),
+			FileSize: info.Size(),
+			Type:     model.UploadTypeImport,
+			UserId:   userID,
+		})
+		if resp.Error != nil {
+			return fmt.Errorf("failed to create upload: %w", resp.Error)
+		}
+
+		printer.PrintT("Upload successfully created, ID: {{.Id}} ", us)
+	}
+
+	finfo, resp := c.UploadData(us.Id, file)
+	if resp.Error != nil {
+		return fmt.Errorf("failed to upload data: %w", resp.Error)
+	}
+
+	printer.PrintT("Import file successfully uploaded, name: {{.Id}}", finfo)
+
+	return nil
+}
+
+func importProcessCmdF(c client.Client, command *cobra.Command, args []string) error {
+	importFile := args[0]
+
+	job, resp := c.CreateJob(&model.Job{
+		Type: model.JOB_TYPE_IMPORT_PROCESS,
+		Data: map[string]string{
+			"import_file": importFile,
+		},
+	})
+	if resp.Error != nil {
+		return fmt.Errorf("failed to create import process job: %w", resp.Error)
+	}
+
+	printer.PrintT("Import process job successfully created, ID: {{.Id}}", job)
+
+	return nil
+}
+
+func importListJobsCmdF(c client.Client, command *cobra.Command, args []string) error {
+	var jobs []*model.Job
+	var resp *model.Response
+	if len(args) == 1 {
+		var job *model.Job
+		job, resp = c.GetJob(args[0])
+		if resp.Error != nil {
+			return fmt.Errorf("failed to get import job: %w", resp.Error)
+		}
+		jobs = append(jobs, job)
+	} else {
+		numJobs, _ := command.Flags().GetInt("limit")
+		jobs, resp = c.GetJobsByType(model.JOB_TYPE_IMPORT_PROCESS, 0, numJobs)
+		if resp.Error != nil {
+			return fmt.Errorf("failed to get import jobs: %w", resp.Error)
+		}
+
+		if len(jobs) == 0 {
+			printer.Print("No import jobs found")
+			return nil
+		}
+	}
+
+	for _, job := range jobs {
+		printer.PrintT(fmt.Sprintf("  ID: {{.Id}}\n  Status: {{.Status}}\n  Created: %s\n  Started: %s\n",
```

Comment:
nit: If we are to split the two use cases into two commands, the template can be shared or we can even create a `printJob` helper for this